Node-DC-EIS-client/process_time_based_output.py

```diff
@@ -231,6 +228,10 @@ def post_process(temp_log,output_file,results_dir,interval,memlogfile,no_graph):
   processed_file.flush()
 
   if not no_graph:
+    import matplotlib
+    matplotlib.use(matplotlib.get_backend())
```

Comment:
Out of curiosity, why are setting the backend to the current backend? Does it reset/fix a bug somewhere?

Reply:
Yes, it was throwing an error on few machines saying the backend display environment has not been set.

Reply:
Does this look okay to merge?